core/src/main/scala/com/mesosphere/package.scala

```diff
@@ -0,0 +1,78 @@
+package com
+
+import com.typesafe.scalalogging.CanLog
+import com.typesafe.scalalogging.LoggerTakingImplicit
+import org.slf4j.MDC
+import scala.language.implicitConversions
+
+package object mesosphere {
```

Comment:
would be nice to have some high level docs desacribing on how to use this logger (basically what's inside `Scheduler`)